docker-test.sh

```diff
@@ -2,89 +2,8 @@
 
 export DOCKER_IMAGE=${DOCKER_IMAGE-orbs}
 export DOCKER_TAG=${DOCKER_TAG-$(git rev-parse --abbrev-ref HEAD | sed -e 's/\//-/g')}
-export NODE_CONFIG_PATH=/opt/orbs/config/topology
-export GOSSIP_PEERS=ws://172.2.1.2:60001,ws://172.2.1.3:60001,ws://172.2.1.4:60001,ws://172.2.1.5:60001,ws://172.2.1.6:60001,ws://172.2.1.7:60001
 
-echo "RUNNING UNIT TESTS"
-docker run --rm -ti \
-    -w /opt/orbs/projects/libs/core-library-typescript \
-    $DOCKER_IMAGE:$DOCKER_TAG yarn test
+docker network create public-api-external-network --subnet 172.2.2.0/24 || true
+docker run -ti --rm --privileged  --network=public-api-external-network --ip 172.2.2.9 -e PREEXISTING_PUBLIC_API_SUBNET=172.2.2 -e CONNECT_FROM_HOST=false -e DOCKER_IMAGE -e DOCKER_TAG -v /Users/idan/dev/orbs/orbs-network/e2e/src:/opt/orbs/e2e/src -v /var/run/docker.sock:/var/run/docker.sock $DOCKER_IMAGE:$DOCKER_TAG  bash -c 'apk add --no-cache docker py-pip && pip install docker-compose && cd e2e && ./build.sh && ./test.sh'
```

Comment:
I suggest you split it in multiple lines so we could maintain it more easily.

Also, instead of `/Users/idan/dev/orbs/orbs-network/e2e/src:/opt/orbs/e2e/src`:
```
`pwd`/e2e/src:/opt/orbs/e2e/src
```

Reply:
I fixed it in another newer branch awaiting pull request